packages/esm-patient-registration-app/src/patient-registration/patient-registration.test.tsx

```diff
@@ -407,6 +517,186 @@ describe('Registering a new patient', () => {
   });
 });
 
+describe('Import an MPI patient record', () => {
```

Comment:
@ibacher Unfortunately, this test isn't providing much value at the moment. The issue seems to be that I'm unable to update the initially mocked values that are supposed to be updated via the useEffect [hook](https://github.com/openmrs/openmrs-esm-patient-management/pull/1397/files#diff-25dfbd3829e19bcd1e08cd0d0e2a572e26c7ace2f93409574bd182379c947497R70).